app/models/genre.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Genre < ApplicationRecord
+  belongs_to :movie
+  validates :genre_type, presence: true
```

Comment:
Can we make a list of genre_types instead of allowing any kind of text?
Let's make an enum of genre types

Reply:
okay apu , how many elements should i keep in the list ?

Reply:
maybe the most common genres, upto you or you can take reference from online

Reply:
done apu